src/app/badges/[id]/page.tsx

```diff
@@ -1,19 +1,59 @@
 'use client';
 
+import OwnershipPill from '@/components/Badges/OwnershipPill';
+import SwipeUpDrawer from '@/components/Badges/SwipeUpDrawer';
 import { GAME_ID } from '@/constants/gameId';
 import { useClientCheck } from '@/hooks/useClientCheck';
 import { useGameState } from '@/hooks/useGameState';
-import React, { useMemo } from 'react';
+import { Stack } from '@mui/material';
+import Box from '@mui/material/Box';
+import Typography from '@mui/material/Typography';
+import Image from 'next/image';
+import Link from 'next/link';
+import { useMemo } from 'react';
 import { useAccount } from 'wagmi';
 
-type Props = {
+interface Props {
```

Comment:
why interface?

Reply:
that was by accident, I have a commit that switches it back. type is superior